### PR TITLE
Feature/IIWA-37

### PR DESCRIPTION
**Pull request by kirbur**
Cards and Titles are made

### CODE REVIEW

File: src/frontend-admin/src/components/reusables/Card/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import styles from './Card.module.css'
+import {Title} from '../Title/Title' 
+
+interface ICardProps {
+    title: any
```
Comment (mtleonard120) [inline src/frontend-admin/src/components/reusables/Card/Card.tsx]: Nice, I agree this should be an `any`. That way we can pass in strings, components, whatever

Comment (mtleonard120) [reply src/frontend-admin/src/components/reusables/Card/Card.tsx]: That said, this likely should be `title?: any`. That'll give us flexibility to have cards without titles if we ever need them.

File: src/frontend-admin/src/components/reusables/Card/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    children: any //any so that dropdown can be used
+}
+
+export const Card: React.FC<ICardProps> = (props) => {
+    const { title, children } = props
+
+    return (
+        <div className={styles.cardMain}>
+            <Title title={title}/>
```
Comment (mtleonard120) [inline src/frontend-admin/src/components/reusables/Card/Card.tsx]: If our title prop is optional, you can change this line to be `{title && <Title title={title} />}` Conditional rendering like that is probably my favorite aspect of jsx :)

File: src/frontend-admin/src/components/reusables/Card/Card.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import styles from './Card.module.css'
+import {Title} from '../Title/Title' 
+
+interface ICardProps {
+    title: any
+    children: any //any so that dropdown can be used
+}
```
Comment (mtleonard120) [inline src/frontend-admin/src/components/reusables/Card/Card.tsx:11]: One other prop you may consider adding here (and really any widely reusable component) is an optional `className`. That way should we ever need to override any stylings when using a card we can easily do so. You can then use your classnames package to concatenate this optional prop with `styles.cardMain`.

**Review by mtleonard120 — CHANGES_REQUESTED** [review]
Just a couple small suggestions - looking good :)

**Review by mcdowellalex — APPROVED** [review]

**Pull request merged**